content/blog/2019/custom-ingress-gateway/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Deploy a custom ingress gateway using cert-manager
+description: Describes how to deploy a custom ingress gateway using cert-manager manually.
+subtitle: Custom ingress gateway
+publishdate: 2019-01-08
+weight: 89
+keywords: [ingress,traffic-management]
+attribution: Julien Senon
+---
+
+This post provides instructions to manually create a custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
```

Comment:
would this be more appropriate as a task in istio.io?  This seems something we want to keep it as a living doc.

Reply:
This started out as a blog, then we moved it to a task, and now it's back to a blog. @frankbu and @vadimeisenbergibm have been involved in the discussion.

I don't care either way, I'll be happy either way.

Reply:
As I said in the previous PR, this should not be a task. If we want it in examples, then it needs more work, but as is it's best as a blog.

Reply:
As much as I like this in a task, I do agree that it needs more work for task. :).